src/ompl/tools/thunder/Thunder.h

```diff
@@ -161,6 +171,26 @@ namespace ompl
             /** \brief Print information about the current setup */
             void print(std::ostream &out = std::cout) const override;
 
+            /** \brief set the from scratch planner to be CForest */
+            void setCforest() {
+              plan_with_cforest_ = true;
+            }
+
+            /** \brief set the from scratch planner to be RRT */
+            void setRRT() {
+              plan_with_cforest_ = false;
+            }
```

Comment:
the implementations of these two functions aren't the same. why this asymmetric and counterintuitive pattern?

Reply:
what would be better here - a bool for each type? or should i create an enum value here? we use this to determine which planner type to make in setup() so i can't store the name of the planner or just create a planner type on these set functions, we also use it to determine the size of the planner ptr vec

Reply:
i'd suggest enums because there are two (maybe more in the future) named choices that are mutually exclusive as suggested by the setter names

if they aren't mutually exclusive and cforest is merely an addon option for RRT that's always used, then we keep the bool and rename the setters.

Reply:
ok - added an enum thunderPlanner, was trying to keep style similar to what's in OMPL already and half copied this from OptimalPlanning.cpp which names an optimalPlanner enum type